src/components/Select/SingleSelect.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import { Icon } from "@/components";
+import { Select, SelectProps } from "./SingleSelect";
+import { Preview } from "@storybook/react";
+interface Props extends SelectProps {
+  clickableNoData?: boolean;
+  showSearch?: boolean;
+}
+const SelectExample = ({ clickableNoData, value, showSearch, ...props }: Props) => {
+  return (
+    <Select
+      value={value}
+      onCreateOption={
+        clickableNoData ? search => console.log("Clicked ", search) : undefined
+      }
+      {...props}
+    >
+      <Select.Trigger />
```

Comment:
I think we know what the `Trigger` for the select is, the user shouldn't need to specify this or the content.

Let's simplify the interface.
```js
<Select value={value} label='label' open={open} multiple>
  <Select.Option value='value1' icon="chart"> Label 1</Select.Option>
   <Select.Option value='value2' icon="database"> Label 2</Select.Option>
<\Select>
```

Reply:
Added trigger as separate inorder to have contextMenu and tooltip only specific to the trigger and not the rest of the elements

Reply:
IMO we shouldn't have a context menu or a tooltip on the select input cc @gjones, but if we need to we can export two interface, one as above and then the lower level componets like the following:

```js
<Select.Container {...props}>
  <Select.Trigger />
  <Select.Content>
  ..... 
  </Select.Content>
</Select.Container>
```